vms/avm/index_test.go

```diff
@@ -53,12 +46,7 @@ func TestIndexTransaction_Ordered(t *testing.T) {
 		tx := buildTX(env.vm.ctx.XChainID, utxoID, txAssetID, addr)
 		require.NoError(tx.SignSECP256K1Fx(env.vm.parser.Codec(), [][]*secp256k1.PrivateKey{{key}}))
 
-		env.vm.ctx.Lock.Unlock()
```

Comment:
Shouldn't `env.vm.state.AddUTXO(utxo)` have the lock held when we are modifying it?